docs/pages/filterable-list-control-panels.md

```diff
@@ -9,22 +9,152 @@ description: The filter control panel allows users to whittle down number of
   parameters, housed in an expandable above the list.
 variation_groups:
   - variations:
-      - variation_code_snippet: ""
-        variation_description: >-
-          ### Collapsed
-
-
-          ![Filterable list collapsed](/design-system/images/uploads/filterable_list_control_panels_collapsed_071620.png)
-
-
-          ### Expanded
-
-
-          ![Filterable list expanded](/design-system/images/uploads/filterable_list_control_panels_expanded_071620.png)
+      - variation_code_snippet: >-
+          <div id="o-filterable-list-controls"
+          class="o-filterable-list-controls">
+              <div class="o-expandable
+                  o-expandable__padded
+                  o-expandable__background
+                  o-expandable__border">
+                  <button class="o-expandable_header o-expandable_target o-expandable_target__expanded" type="button">
+                      <span class="h4 o-expandable_header-left o-expandable_label">
+                      Filter posts
+                      </span>
+                      <span class="o-expandable_header-right o-expandable_link">
+                          <span class="o-expandable_cue o-expandable_cue-open">
+                              <span class="u-visually-hidden-on-mobile
+                                  ">
+                              Show
+                              filters
+                              </span>
+                              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17 20.4" class="cf-icon-svg">
+                                  <path d="M16.416 10.283A7.917 7.917 0 1 1 8.5 2.366a7.916 7.916 0 0 1 7.916 7.917zm-2.958.01a.792.792 0 0 0-.792-.792H9.284V6.12a.792.792 0 1 0-1.583 0V9.5H4.32a.792.792 0 0 0 0 1.584H7.7v3.382a.792.792 0 0 0 1.583 0v-3.382h3.382a.792.792 0 0 0 .792-.791z"></path>
+                              </svg>
+                          </span>
+                          <span class="o-expandable_cue o-expandable_cue-close">
+                              <span class="u-visually-hidden-on-mobile
+                                  ">
+                              Hide
+                              filters
+                              </span>
+                              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17 20.4" class="cf-icon-svg">
+                                  <path d="M16.416 10.283A7.917 7.917 0 1 1 8.5 2.366a7.916 7.916 0 0 1 7.916 7.917zm-2.958.01a.792.792 0 0 0-.792-.792H4.32a.792.792 0 0 0 0 1.583h8.346a.792.792 0 0 0 .792-.791z"></path>
+                              </svg>
+                          </span>
+                      </span>
+                  </button>
+                  <div class="o-expandable_content o-expandable_content__transition u-is-animating o-expandable_content__expanded" style="max-height: 398px;">
```

Comment:
I'm seeing some double expandable buttons w/ the PR preview. Try replacing these classes with `o-expandable_content o-expandable_content__onload-open` to [open the expandable on load](https://deploy-preview-1334--cfpb-design-system.netlify.app/design-system/components/expandables#standard-expandable-open-on-load).

<img width="791" alt="Screen Shot 2021-11-08 at 12 29 53 PM" src="https://user-images.githubusercontent.com/1060248/140789861-e274f498-8c6b-490b-8d0e-142703dff858.png">

Reply:
Fixed, thanks!